libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -130,6 +130,13 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     AP_GROUPINFO("_ALT_TYPE", 10, AP_Follow, _alt_type, AP_FOLLOW_ALT_TYPE_DEFAULT),
 #endif
 
+    // @Param: _OPTIONS
+    // @DisplayName: Follow options
+    // @Description: Follow options bitmask. Specifies mount behaviour in follow mode
```

Comment:
```suggestion
    // @Description: Follow options bitmask.
```

may be used for other things later